libdnf/dnf-context.cpp

```diff
@@ -1536,6 +1553,22 @@ dnf_context_set_keep_cache(DnfContext *context, gboolean keep_cache)
     mainConf.keepcache().set(libdnf::Option::Priority::RUNTIME, keep_cache);
 }
 
+/**
+ * dnf_context_set_ignore_arch:
+ * @context: a #DnfContext instance.
+ * @ignorearch: %TRUE allow installation of architecture incompatible packages
+ *
+ * Enables or disables ignoring CPU architecture for package installation.
+ *
+ * Since: 0.1.0
```

Comment:
This needs to be a future version of libdnf.

Reply:
In the other PR, they also bumped VERSION.cmake together with the change. Is that something you would suggest here as well or is that something that would happen after this in a separate commit?

Reply:
From my limited experience, same PR, different commit (ideally, second). If you need to apply this patch to an rpm/backport you can drop the second commit which only does version bumps. If there's less worse ways of doing this, I'm keen to learn also.

Reply:
ok, pointed at current + 0.0.1 and added a 2nd commit doing the bump